src/test/java/com/revature/controllertests/RequestControllerTest.java

```diff
@@ -88,7 +88,7 @@ public void givenValidDataCreateRequest() throws Exception {
 				.contentType(MediaType.APPLICATION_JSON_VALUE)
 				.content(objMapper.writeValueAsString(rc.requestToJSONRequest(request))));
 
-		JSONRequest ret = objMapper.readValue(result.andReturn().getResponse()
+		objMapper.readValue(result.andReturn().getResponse()
```

Comment:
is this even needed in this test at all?

Reply:
I did not make that change, but it looks like it is not needed I will go change it.